cli/src/commands/get/comments.rs

```diff
@@ -289,42 +291,69 @@ fn get_comments_from_uids(
 
             statistics.add_comments(page.len());
 
-            match &model_version {
-                None => {
-                    let annotations = client
-                        .get_labellings(&dataset_name, page.iter().map(|comment| &comment.uid))
-                        .context("Operation to get labellings has failed.")?;
-                    let comments = page.into_iter().zip(annotations.into_iter()).map(
-                        |(comment, mut annotated)| {
-                            if !include_predictions {
-                                annotated = annotated.without_predictions();
-                            }
-                            annotated.comment = comment;
-                            if annotated.has_annotations() {
-                                statistics.add_annotated(1);
-                            }
-                            annotated
-                        },
-                    );
-                    print_resources_as_json(comments, &mut writer)
-                }
-                Some(model_version) => {
-                    let predictions = client
-                        .get_predictions(
-                            &dataset_name,
-                            &ModelVersion(model_version.to_string()),
-                            page.iter().map(|comment| &comment.uid),
-                        )
-                        .context("Operation to get predictions has failed.")?;
-                    let comments = page.into_iter().zip(predictions.into_iter()).map(
-                        |(comment, prediction)| AutoThresholdedComment {
+            if let Some(model_version) = &model_version {
+                let model_version = model_version
+                    .parse::<u32>()
+                    .context("Invalid model version")?;
+                let predictions = client
+                    .get_comment_predictions(
+                        &dataset_name,
+                        &ModelVersion(model_version),
+                        page.iter().map(|comment| &comment.uid),
+                    )
+                    .context("Operation to get predictions has failed.")?;
+                let comments =
+                    page.into_iter()
+                        .zip(predictions.into_iter())
+                        .map(|(comment, prediction)| AnnotatedComment {
                             comment: comment,
-                            labelling: prediction.labels,
-                            entities: prediction.entities,
-                        },
-                    );
-                    print_resources_as_json(comments, &mut writer)
-                }
+                            labelling: Some(vec![Labelling {
+                                group: DEFAULT_LABEL_GROUP_NAME.clone(),
+                                assigned: Vec::new(),
+                                dismissed: Vec::new(),
```

Comment:
I would add a comment here explaining why you are not populating these fields: because the `predict_comments` endpoint does not return them but we still want the shape of the returned data to be consistent across the different uses of the CLI tool.

Same below for entities.